Add python-graphviz in dependencies, add oualib maintainer

FernandezMathieu opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

FernandezMathieu commented:
Hi  @oualib

I added you maintainer of this project to let you upgrade the VerticaPy version in conda-forge. Tell me if you need to learn how conda-forge recipe works.
We don't have the time to maintain this recipe in the future. This point will be treated in our next meeting.

FYI, a few of our users faced issues with the conda-forge installation of VerticaPy when they were using the `plot_tree` method, they got this error :` ModuleNotFoundError: No module named 'graphviz'`.
They tried to install "graphviz" with conda to resolve the problem, but it doesn't work.

We found that conda forge `conda install graphviz" will install the Graphviz software instead of the python librarie.
So, i added "python-graphviz" in the dependencies part of the meta file to fix the issue.

Regards.

FernandezMathieu commented:
@PDataScienceTeam

FernandezMathieu commented:
@oualib @afard after mergin this, you will be maintainers.
